dataprofiler/tests/profilers/test_column_profile_compilers.py

```diff
@@ -87,77 +87,6 @@ def test_add_profilers(self):
         self.assertEqual(3, merged_compiler._profiles["test"])
         self.assertEqual("compiler1", merged_compiler.name)
 
-    @mock.patch("dataprofiler.profilers.data_labeler_column_profile.DataLabeler")
```

Comment:
moving to its own `TestColumnDataLabelerCompiler` test class

Reply:
nice